src/core/core.scale.js

```diff
@@ -523,6 +523,10 @@ module.exports = Element.extend({
 		if ((typeof rawValue === 'number' || rawValue instanceof Number) && !isFinite(rawValue)) {
 			return NaN;
 		}
+		// Float-bar support. Handling arrays
```

Comment:
Please remove that comment, the Scale class shouldn't be aware of "float bar"

Reply:
done